iceddev/iggins
# Predicates

#### What's this PR do?

Implementation of Iggins keypress functionality.  An Irken application exposes 'keypress' allowing other parts of the application to register a function to be handled on 'keydown'.
#### Where should the reviewer start?

Clone this repo
Go to the directory of your Irken app and `npm install git://github.com/iceddev/iggins'
Link to the local files with`npm link <relative-path-to-Iggins-clone>`
Register the plugin in your Irken app
#### How should this be manually tested?

Register a keypress in your application.  Identify a function that needs to be associated with a keypress. In the component where the function is invoked enter the following:
`app.keypress(app.keypress.KEY_STROKE, yourHandler);
rebuild your Irken app
verify that your key mapping is working.
#### What are the relevant tickets?

This functionality was created in regards to [issue #88](https://github.com/parallaxinc/ChromeIDE/issues/88) on ChromeIDE
